dags/pytorch_xla/pytorchxla_torchbench.py

```diff
@@ -14,37 +14,113 @@
 
 """A DAG to run all TorchBench tests with nightly version."""
 
-import datetime
 from airflow import models
+import datetime
 from dags import composer_env
 from dags.pytorch_xla.configs import pytorchxla_torchbench_config as config
-from dags.vm_resource import TpuVersion, Zone
+import dags.vm_resource as resource
 
-_MODELS = [
-    "BERT_pytorch",
-    "Background_Matting",
-    "hf_Bert",
-    "Super_SloMo",
-    "tts_angular",
-]
+# Schudule the job to run once per two days at 5:00PM UTC.
+SCHEDULED_TIME = "0 17 */2 * *" if composer_env.is_prod_env() else None
 
-# Schudule the job to run every day at 5:00PM UTC.
-SCHEDULED_TIME = "0 17 * * *" if composer_env.is_prod_env() else None
 
 with models.DAG(
     dag_id="pytorchxla-torchbench",
     schedule=SCHEDULED_TIME,
     tags=["pytorchxla", "nightly", "torchbench"],
-    start_date=datetime.datetime(2023, 8, 29),
+    start_date=datetime.datetime(2024, 1, 1),
     catchup=False,
 ) as dag:
-  for model in _MODELS:
-    torchbench_extra_flags = [f"--filter={model}"]
-    config.get_torchbench_config(
-        tpu_version=TpuVersion.V4,
-        tpu_cores=8,
-        tpu_zone=Zone.US_CENTRAL2_B.value,
-        model_name=model,
-        time_out_in_min=60,
-        extraFlags=" ".join(torchbench_extra_flags),
-    ).run()
+  model = "all"
+  torchbench_extra_flags = [f"--filter={model}"]
+  # Running on V4-8:
+  config.get_torchbench_tpu_config(
+      tpu_version=resource.TpuVersion.V4,
+      tpu_cores=8,
+      project=resource.Project.CLOUD_ML_AUTO_SOLUTIONS,
+      tpu_zone=resource.Zone.US_CENTRAL2_B,
+      runtime_version=resource.RuntimeVersion.TPU_UBUNTU2204_BASE,
+      model_name=model,
+      time_out_in_min=1600,
+      extraFlags=" ".join(torchbench_extra_flags),
+  ).run()
+
+  # Running on V5P
+  config.get_torchbench_tpu_config(
+      tpu_version=resource.TpuVersion.V5P,
+      tpu_cores=8,
+      project=resource.Project.TPU_PROD_ENV_AUTOMATED,
+      tpu_zone=resource.Zone.US_EAST5_A,
+      runtime_version=resource.RuntimeVersion.V2_ALPHA_TPUV5,
+      network=resource.V5_NETWORKS,
+      subnetwork=resource.V5E_SUBNETWORKS,
```

Comment:
This should be `V5P_SUBNETWORKS`

Reply:
V5E network still works on v5p platforms as I just tested, but I will update to v5p network to be consistent.